### PR TITLE
Refactor test cases code in InstructorFeedbackResultsPageUiTest and StudentFeedbackResultsPageUiTest #6410

### LINKED ISSUE #6410: Refactor test cases code in InstructorFeedbackResultsPageUiTest and StudentFeedbackResultsPageUiTest
Currently, the testing code of toggle additional question info ([More] and [Less]) is confusing as show below.
``` java
assertEquals("[more]", resultsPage.getQuestionAdditionalInfoButtonText(8, "section-1-giver-1-recipient-1"));
assertTrue(resultsPage.clickQuestionAdditionalInfoButton(8, "section-1-giver-1-recipient-1"));
assertEquals("[less]", resultsPage.getQuestionAdditionalInfoButtonText(8, "section-1-giver-1-recipient-1"));
assertFalse(resultsPage.clickQuestionAdditionalInfoButton(8, "section-1-giver-1-recipient-1"));
assertEquals("[more]", resultsPage.getQuestionAdditionalInfoButtonText(8, "section-1-giver-1-recipient-1"));
```
`clickQuestionAdditionalInfoButton` should just click the button rather than clicking button and returning whether the additional info panel is visible or not.

Refer to #6352 for discussion.

### PR DESCRIPTION
**Pull request by xpdavid**
Fixes #6410

### CODE REVIEW

**Comment by xpdavid** [comment]
Ready for review.

**Comment by xpdavid** [comment]
@chowyb Yes. Initially I have the checking for `[more]`, but it is actually redundant and I remove it as the checking for `[more]` is actually verified in the html verification :P (actually before/after every checking for the toggling of `[more]` or `[less]`, there is a html verification)

File: src/test/resources/pages/instructorFeedbackResultsSortGiverRecipientQuestionTeam.html [hunk 1]
```diff
@@ -1939,7 +1939,7 @@
                                   [more]
                                 </a>
                                 <br>
-                                <span id="questionAdditionalInfo-8-section-1-giver-1-recipient-1" style="display: none;">
+                                <span id="questionAdditionalInfo-8-section-1-giver-1-recipient-1" style="display:none;">
```
Comment (chowyb) [inline src/test/resources/pages/instructorFeedbackResultsSortGiverRecipientQuestionTeam.html]: Probably should revert this change if possible - the missing space does not seem consistent with the rest of the file so I'm assuming it's unintentional, although if this is not possible (eg. it was caused by JS modifying the DOM and causes test failures if changed) then it should be fine to leave this change here.

Comment (xpdavid) [inline src/test/resources/pages/instructorFeedbackResultsSortGiverRecipientQuestionTeam.html]: @chowyb Note that in this commit dded058 I actually swap the checking and I run god mode to change the `html` file or the test case will fail.
The missing space is actually widely existing in the whole project as `feedbackQuestionAdditionalInfoTemplate.html` and `feedbackQuestionContribAdditionalInfoTemplate.html` don't follow the convention
``` html
&nbsp;<span style=" white-space: normal;">
    <a href="javascript:;" id="questionAdditionalInfoButton-${questionNumber}-${additionalInfoId}" class="color_gray" onclick="toggleAdditionalQuestionInfo('${questionNumber}-${additionalInfoId}')" data-more="${more}" data-less="${less}">${more}</a>
    <br>
    <span id="questionAdditionalInfo-${questionNumber}-${additionalInfoId}" style="display:none;">${questionAdditionalInfo}</span>
</span>
```
``` html
&nbsp;<span style=" white-space: normal;">
    <a href="javascript:;" id="questionAdditionalInfoButton-${questionNumber}-${additionalInfoId}" class="small link-in-dark-bg" onclick="toggleAdditionalQuestionInfo('${questionNumber}-${additionalInfoId}')" data-more="${more}" data-less="${less}">${more}</a>
    <br>
    <span id="questionAdditionalInfo-${questionNumber}-${additionalInfoId}" style="display:none;">${questionAdditionalInfo}</span>
</span>
```
When JS modifying the DOM, it actually **add** a whitespace (this is why when the comparison is done **after** the checking of toggling, it got whitespace)

Comment (chowyb) [inline src/test/resources/pages/instructorFeedbackResultsSortGiverRecipientQuestionTeam.html]: Alright, everything looks good then 👍

File: src/test/java/teammates/test/cases/browsertests/StudentFeedbackResultsPageUiTest.java [hunk 2]
```diff
@@ -180,4 +176,17 @@ private StudentFeedbackResultsPage loginToStudentFeedbackResultsPage(String stud
...
+        resultsPage.clickQuestionAdditionalInfoButton(qnNumber, additionalInfoId);
+        assertFalse(resultsPage.isQuestionAdditionalInfoVisible(qnNumber, additionalInfoId));
+        assertEquals("[more]", resultsPage.getQuestionAdditionalInfoButtonText(qnNumber, additionalInfoId));
+    }
+    
+    private void verifyQuestionAdditionalInfoExpand(int qnNumber, String additionalInfoId) {
+        resultsPage.clickQuestionAdditionalInfoButton(qnNumber, additionalInfoId);
+        assertTrue(resultsPage.isQuestionAdditionalInfoVisible(qnNumber, additionalInfoId));
+        assertEquals("[less]", resultsPage.getQuestionAdditionalInfoButtonText(qnNumber, additionalInfoId));
+    }
```
Comment (damithc) [inline src/test/java/teammates/test/cases/browsertests/StudentFeedbackResultsPageUiTest.java]: These methods are duplicated in the other test class? If so, any chance of reducing the duplication?

Comment (xpdavid) [inline src/test/java/teammates/test/cases/browsertests/StudentFeedbackResultsPageUiTest.java]: These methods (the checking for [less] or [more] in question additional info) are only used in `InstructorFeedbackResultsPageUiTest` and `StudentFeedbackResultsPageUiTest`. To generalise them into a common method may be overkilled.

**Pull request merged**